tests/integrations/test_postcode_lookup_helper.py

```diff
@@ -73,14 +74,185 @@ def test_get_addresses_from_postcode_handles_500_status_code(mock_get, faker):
             get_addresses_from_postcode(postcode)
 
 
+def test_address_builder_given_house_name_address_returns_address():
+    # GIVEN
+    lpi_info = {
+        'PAO_TEXT': 'The Vicarage',
+        'STREET_DESCRIPTION': 'Church Lane'
+    }
+
+    # WHEN
+    address = address_builder(lpi_info)
+
+    # THEN
+    assert address['building_and_street_line_1'] == 'The Vicarage'
+    assert address['building_and_street_line_2'] == 'Church Lane'
+
+
+def test_address_builder_given_house_number_style_address_returns_address():
+    # GIVEN
+    lpi_info = {
+        'PAO_START_NUMBER': 12,
+        'STREET_DESCRIPTION': 'Church Lane'
+    }
+
+    # WHEN
+    address = address_builder(lpi_info)
+
+    # THEN
+    assert address['building_and_street_line_1'] == '12 Church Lane'
+    assert address['building_and_street_line_2'] == ''
+
+
+def test_address_builder_given_flat_number_style_address_returns_address():
+    # GIVEN
+    lpi_info = {
+        'PAO_START_NUMBER': 12,
+        'PAO_START_SUFFIX': 'B',
+        'STREET_DESCRIPTION': 'Church Lane'
+    }
+
+    # WHEN
+    address = address_builder(lpi_info)
+
+    # THEN
+    assert address['building_and_street_line_1'] == '12B Church Lane'
+    assert address['building_and_street_line_2'] == ''
+
+
+def test_address_builder_given_house_range_number_style_address_returns_address():
+    # GIVEN
+    lpi_info = {
+        'PAO_START_NUMBER': 12,
+        'PAO_END_NUMBER': 14,
+        'STREET_DESCRIPTION': 'Church Lane'
+    }
+
+    # WHEN
+    address = address_builder(lpi_info)
+
+    # THEN
+    assert address['building_and_street_line_1'] == '12-14 Church Lane'
+    assert address['building_and_street_line_2'] == ''
+
+
+def test_address_builder_given_secondary_style_address_returns_address():
+    # GIVEN
+    lpi_info = {
+        'SAO_START_NUMBER': 12,
+        'PAO_TEXT': 'Sussex Court',
+        'STREET_DESCRIPTION': 'Church Lane'
+    }
+
+    # WHEN
+    address = address_builder(lpi_info)
+
+    # THEN
+    assert address['building_and_street_line_1'] == '12, Sussex Court'
+    assert address['building_and_street_line_2'] == 'Church Lane'
+
+
+def test_address_builder_given_one_line_address_and_org_returns_org_as_line_1():
+    # GIVEN
```

Comment:
Are we going to put comments in for given when then for all tests going forward, more of consistency thing as no other tests have them?

Reply:
your call
I use them to structure my thinking and make it clearer for readers what I am up to 
I know they aren't for everyone

Reply:
I personally think they are fine but for the sake of consistency lets remove them to make it like the rest of the tests

Reply:
removed